src/features/events/components/EventActionButtons.tsx

```diff
@@ -78,15 +78,15 @@ const EventActionButtons: React.FunctionComponent<EventActionButtonsProps> = ({
   };
 
   const handleOnCampaignSelected = async (campaignId: number) => {
-    await updateEvent({ campaign_id: campaignId });
+    const updatedEvent = await updateEvent({ campaign_id: campaignId });
     await router.push(
       `/organize/${orgId}/projects/${campaignId}/events/${event.id}`
     );
     showSnackbar(
       'success',
       messages.eventChangeCampaignDialog.success({
-        campaignTitle: event.campaign!.title,
-        eventTitle: event.title!,
+        campaignTitle: updatedEvent.campaign?.title || '?',
```

Comment:
It's not possible to move an event to "no project", so we know that the campaign.title will always exist. We can therefore default to an empty string. 

I am not sure if defaulting to a string containing an `?` is common? But we usually default to empty strings when we know that it's safe to do so.